src/pages/RouteDetails/components/Map/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+import { MapContainer, TileLayer } from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+
+import RoutingComponent from './RoutingComponent';
+import './styles.css';
+
+const Map: FC = () => {
+  const locations = ['New York', 'Washington'];
```

Comment:
why we have it? if its temporary left comment. in other case - delete hardcoded values

Reply:
done